tests/tests/test_bson.py

```diff
@@ -85,3 +85,22 @@ def test_read_bson(
                 assert len(row) == 5
                 assert row["beatle_name"] in beatles
                 assert beatles.index(row["beatle_name"]) == row["beatle_idx"] - 1
+
+def test_null_handling(
+    glaredb_connection: psycopg2.extensions.connection,
+    tmp_path_factory: pytest.TempPathFactory,
+):
+    tmp_dir = tmp_path_factory.mktemp(basename="null_handling", numbered=True)
+    data_path = tmp_dir.joinpath("mixed.bson")
+    
+    with open(data_path, "wb") as f:
+        for i in range(100):
+            f.write(bson.encode({"a": 1}))
+            
+        for i in range(10):
+            f.write(bson.encode({"a": None}))
+
+    with glaredb_connection.cursor() as curr:
+        curr.execute(f"select count(*) from '{data_path}'")
```

Comment:
the `count(*)` query was previously working, We should explicitly check the values

Reply:
I ran this test without the patch and it failed. 

Because this uses the streaming bson table/file code, which uses the same bson-to-recordbatch as mongodb, we would expect the count to break here too.